atest/testdata/standard_libraries/builtin/repeat_keyword.robot

```diff
@@ -89,3 +96,8 @@ First Continuable Failure And Then Pass Execution
     Pass Execution If    "${PASS EXECUTION}" == "yes"    Message
     Set Suite Variable    ${PASS EXECUTION}    yes
     Run Keyword And Continue On Failure    Fail    Continuable
+
+Keyword Executing For 1 second
+    [Arguments]     ${TIME FORMAT}
+    Log    This is done for ${TIME FORMAT}
+    Sleep    1 seconds
```

Comment:
Related to the above comment, 1 second is too long. I don't actually see why sleeping is needed altogether.

Also, we want to have newlines at the end of files. I recommend configuring your editor/IDE to add them automatically.